api-client/src/endpoints/submitHelpForm.ts

```diff
@@ -1,15 +1,142 @@
 import OfferModel from '../models/OfferModel'
```

Comment:
I would stick to one naming if possible, so renaming this to `submitMalteHelpForm`